test-cases/hash-bytes/lib.rs

```diff
@@ -0,0 +1,78 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
+
+#[ink::contract]
+mod hash_bytes {
+    use ink::env::hash::Blake2x256;
+
+    #[ink(storage)]
+    pub struct HashBytes {}
+
+    impl HashBytes {
+        #[ink(constructor)]
+        pub fn new() -> Self {
+            Self {}
+        }
+
+        #[ink(message)]
+        pub fn get_hash_bytes(&self, input: [u8; 32]) -> [u8; 32] {
+            self.env().hash_bytes::<Blake2x256>(&input)
+        }
+    }
+
+    impl Default for HashBytes {
+        fn default() -> Self {
+            Self::new()
+        }
+    }
+
+    #[cfg(test)]
+    mod tests {
+        use super::*;
+
+        #[ink::test]
+        fn hash_bytes_works() {
+            // Given
+            let contract = HashBytes::new();
+
+            // When
+            let hash_bytes = contract.get_hash_bytes([0x01; 32]);
+
+            // Then
+            assert!(hash_bytes.len() == 32);
```

Comment:
This is not checking if the hash function works properly. This just checks for the length of the hash_bytes.